trivy.yaml

```diff
@@ -14,10 +14,16 @@ environment:
       - go
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      uri: https://github.com/aquasecurity/trivy/archive/v${{package.version}}/trivy-${{package.version}}.tar.gz
-      expected-sha512: a9966dad0c16a12bcfbced45489a41f940bdbe7b4b44ee6845fef63523e84d1cccc20c732681708d4234eda91ca4e98d0e97c8d17041ea066599165e0309aa85
+      repository: https://github.com/aquasecurity/trivy
+      tag: v${{package.version}}
+      expected-commit: ba825b2ae1b0481d4b0d2845eafbb3cc83914c2f
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/crypto@v0.17.0 github.com/containerd/containerd@v1.7.11
+      go-version: "1.21.0"
```

Comment:
I'd suggest to avoid setting the go-version if possible. It is gonna be a bit troublesome to maintain this value in the future.

Reply:
Yeah - it was a copy / paste thing as done quite a lot here https://github.com/search?q=repo%3Awolfi-dev%2Fos%20go-version&type=code.  I seem to remember a discussion recently maybe with @cpanato as we had a similar thought.

I think there was an issue if we set go 1.21 as the default in the go/bump pipeline? https://github.com/wolfi-dev/os/blob/a97e56a0f5e229a5703936ad4b075318f2b0f826/pipelines/go/bump.yaml#L17-L19

Reply:
We should create one issue on gobump to fix it.

Reply:
good call - created https://github.com/chainguard-dev/gobump/issues/5

Reply:
Thanks